pkg/build/buildscheme/scheme.go

```diff
@@ -33,12 +32,12 @@ func init() {
 	legacy.InstallInternalLegacyBuild(annotationDecodingScheme)
 	utilruntime.Must(buildv1helpers.Install(annotationDecodingScheme))
```

Comment:
If you're switching to externals, update this line and the one above to only install externals.  So just buildv1.Install and InstallLegacyBuild

Reply:
not switching yet, reverted